pkg/migration/controllers/migrationschedule.go

```diff
@@ -149,50 +149,58 @@ func (m *MigrationScheduleController) handle(ctx context.Context, migrationSched
 				err.Error())
 			return nil
 		}
-		if migrationSchedule.Spec.AutoSuspend {
-			var remoteMigrSched *stork_api.MigrationSchedule
-			remoteMigrSched, err = remoteOps.GetMigrationSchedule(migrationSchedule.Name, migrationSchedule.Namespace)
-			if errors.IsNotFound(err) {
-				namespace, err := core.Instance().GetNamespace(migrationSchedule.Namespace)
-				if err != nil {
-					return err
-				}
-				namespace.ResourceVersion = ""
-				_, err = coreOps.CreateNamespace(namespace)
-				if err != nil && !errors.IsAlreadyExists(err) {
-					return err
-				}
-				// create new migrationschedule on remote cluster
-				remoteMigrSched = migrationSchedule.DeepCopy()
-				remoteMigrSched.ResourceVersion = ""
-				remoteMigrSched.UID = ""
-				if remoteMigrSched.Annotations == nil {
-					remoteMigrSched.Annotations = make(map[string]string)
-				}
-				remoteMigrSched.Annotations[StorkMigrationScheduleCopied] = "true"
-				suspend := true
-				remoteMigrSched.Spec.Suspend = &suspend
-				remoteMigrSched.Status = stork_api.MigrationScheduleStatus{}
-				if _, err := remoteOps.CreateMigrationSchedule(remoteMigrSched); err != nil {
-					return err
-				}
 
-			} else if err != nil {
+		// create migrationSchedule in remote cluster irrespective of autoSuspend value
```

Comment:
I am missing the point. Help me understand why we need to copy the migrationschedules?

Reply:
perform failover command is run in the context of destination cluster with the flag --migration-reference <migrationScheduleName>
And the migrationSchedule is then used to get the clusterPairName and create last mile migration object. 
Obviously for this to work, the reference migrationschedule should exist in the destination cluster.

As of stork 23.11, migrationschedule's staticCopy is created in the remote cluster only in case the original source migrationSchedule has autoSuspend set to true.
From stork 24.2 onwards, we will always create the staticCopy in the remote cluster.